course/src/main/java/in/testpress/course/util/ExoPlayerUtil.java

```diff
@@ -322,6 +326,27 @@ public void initializePlayer() {
         addPlayPauseOnClickListener();
     }
 
+    private void initializeDoubleClickOverlay() {
+        initializeOverlayPerformListener();
```

Comment:
what is this perform listener? I am still not getting what this initializes by looking at the name?

Reply:
Their internal name is perform listener. But what we are using it for ?

Reply:
Go to that source code and check when that listener will be triggered and come up with good name